packages/discovery-provider/src/api/v1/helpers.py

```diff
@@ -305,6 +305,18 @@ def extend_user(user, current_user_id=None):
     return user
 
 
+def extend_account(account):
+    user = account["user"]
+    # Extract before extend_user as it will delete this
+    playlist_library = user.get("playlist_library", {})
+    return {
+        "user": extend_user(user),
+        "playlist_library": playlist_library,
```

Comment:
Question: The existing endpoint does not encode ids for playlists in `playlist_library` or `playlists`, so I didn't add that here.. Since this will be consumed through SDK, there's no harm in updating the response here. Open to suggestions one way or the other.